### PR TITLE
Landing page: change messaging

### PR DESCRIPTION
**Pull request by richardmuscat**
Making messaging simpler and consistent as per P2 thread.

This change includes:
- A call to action that says “Connect Jetpack” to match with what we’re
asking the user to do
- A heading saying “Please Connect Jetpack”
- A simpler line of descriptive text highlighting the key features
- Centre aligned text and Jump Start style box

### CODE REVIEW

File: views/admin/admin-page.php [hunk 1]
```diff
@@ -56,18 +56,15 @@
 </div><!-- .landing -->
 
 	<?php else : ?>
-		<div class="wpcom-connect">
-			<h1 title="Boost traffic, enhance security, and improve performance."><?php esc_html_e( 'Boost traffic, enhance security, and improve performance.', 'jetpack' ); ?></h1>
+		<div id="jump-start-area" class="j-row">
+			<h1 title="Please Connect Jetpack."><?php esc_html_e( 'Please Connect Jetpack', 'jetpack' ); ?></h1>
```
Comment (dereksmart) [inline views/admin/admin-page.php]: Title attributes should be translatable as well

File: views/admin/admin-page.php [hunk 2]
```diff
@@ -56,18 +56,15 @@
...
 			<div class="j-row">
-				<div class="j-col j-sm-12 j-md-8 j-lrg-7 connect-desc">
-					<p><?php _e('Jetpack connects your site to WordPress.com to give you traffic and customization tools, enhanced security, speed boosts, and more.', 'jetpack' ); ?></p>
-					<p><?php _e('To start using Jetpack, connect to your WordPress.com account by clicking the button (if you don’t have an account you can create one quickly and for free).', 'jetpack' ); ?></p>
-				</div>
-				<div class="j-col j-sm-12 j-md-4 j-lrg-5 connect-btn">
+				<div class="j-col j-sm-12 j-md-8 connect-desc" style="float:none; margin:0 auto; text-align:center; padding-bottom:20px;">
```
Comment (dereksmart) [inline views/admin/admin-page.php]: inline styles?  cc/ @jeffgolenski

File: views/admin/admin-page.php [hunk 3]
```diff
@@ -56,18 +56,15 @@
...
+					<p><?php echo wp_kses( __( 'Connecting Jetpack will show you <strong>stats</strong> about your traffic, <strong>protect</strong> you from brute force attacks, <strong>speed up</strong> your images and photos, and enable other <strong>traffic and security</strong> features.' ), 'jetpack' ) ?></p>
 					<?php if ( ! $data['is_connected'] && current_user_can( 'jetpack_connect' ) ) : ?>
-						<a href="<?php echo Jetpack::init()->build_connect_url() ?>" class="download-jetpack"><?php esc_html_e( 'Connect to WordPress.com', 'jetpack' ); ?></a>
+						<a href="<?php echo Jetpack::init()->build_connect_url() ?>" class="download-jetpack"><?php esc_html_e( 'Connect Jetpack', 'jetpack' ); ?></a>
 					<?php elseif ( $data['is_connected'] && ! $data['is_user_connected'] && current_user_can( 'jetpack_connect_user' ) ) : ?>
-						<a href="<?php echo Jetpack::init()->build_connect_url() ?>" class="download-jetpack"><?php esc_html_e( 'Link your account to WordPress.com', 'jetpack' ); ?></a>
+						<a href="<?php echo Jetpack::init()->build_connect_url() ?>" class="download-jetpack"><?php esc_html_e( 'Link your account to Jetpack', 'jetpack' ); ?></a>
```
Comment (dereksmart) [inline views/admin/admin-page.php]: "Connect Jetpack" and "Link your account to Jetpack" is kind of confusing, as the auth screen is "Jetpack would like to connect to your WordPress.com account.".  Maybe we can match this text up a little better.

File: views/admin/admin-page.php [hunk 4]
```diff
@@ -56,18 +56,15 @@
...
+		<div id="jump-start-area" class="j-row">
+			<h1 title="Please Connect Jetpack."><?php esc_html_e( 'Please Connect Jetpack', 'jetpack' ); ?></h1>
 			<div class="j-row">
-				<div class="j-col j-sm-12 j-md-8 j-lrg-7 connect-desc">
-					<p><?php _e('Jetpack connects your site to WordPress.com to give you traffic and customization tools, enhanced security, speed boosts, and more.', 'jetpack' ); ?></p>
-					<p><?php _e('To start using Jetpack, connect to your WordPress.com account by clicking the button (if you don’t have an account you can create one quickly and for free).', 'jetpack' ); ?></p>
-				</div>
-				<div class="j-col j-sm-12 j-md-4 j-lrg-5 connect-btn">
+				<div class="j-col j-sm-12 j-md-8 connect-desc" style="float:none; margin:0 auto; text-align:center; padding-bottom:20px;">
+					<p><?php echo wp_kses( __( 'Connecting Jetpack will show you <strong>stats</strong> about your traffic, <strong>protect</strong> you from brute force attacks, <strong>speed up</strong> your images and photos, and enable other <strong>traffic and security</strong> features.' ), 'jetpack' ) ?></p>
```
Comment (dereksmart) [inline views/admin/admin-page.php]: i18n prefers the use of `sprintf()` or `printf` for HTML tags in translated strings.

File: views/admin/admin-page.php (hunk 1 above)
Comment (richardmuscat) [inline views/admin/admin-page.php]: Fixed in https://github.com/Automattic/jetpack/commit/e524656a1c2a42ce4937185ace3aaee25cf36248

File: views/admin/admin-page.php (hunk 2 above)
Comment (richardmuscat) [inline views/admin/admin-page.php]: Fixed in https://github.com/Automattic/jetpack/commit/23c4a8cfbcd1a95b64ad5cbe181836129b9839c6

File: views/admin/admin-page.php (hunk 4 above)
Comment (richardmuscat) [inline views/admin/admin-page.php]: Fixed to say "Connect your account" in https://github.com/Automattic/jetpack/commit/02430db4a57dbdc0e754c42d154756fe7f468647

File: views/admin/admin-page.php (hunk 3 above)
Comment (richardmuscat) [inline views/admin/admin-page.php]: Fixed to say "Connect your account" in https://github.com/Automattic/jetpack/commit/02430db4a57dbdc0e754c42d154756fe7f468647

**Comment by zinigor** [comment]
Rebased, the changes look good, needs a design review.

**Comment by jeffgolenski** [comment]
Visual:

screen shot 2015-08-19 at 2 01 47 pm

I think it's a much needed improvement, thanks @richardmuscat In the future we can iterate on this and make it even better, but for now it's grand!

**Comment by dereksmart** [comment]
:shipit:

**Comment by richardmuscat** [comment]
Woot

**Pull request merged**